src/lib.rs

```diff
@@ -592,15 +828,17 @@ impl InflateStream {
                 let (method, info) = (b & 0xF, b >> 4);
                 debug!("ZLIB CM=0x{:x} CINFO=0x{:x}", method, info);
                 match method {
-                    8 => {/* DEFLATE */}
-                    _ => return Err(format!("unknown ZLIB method CM=0x{:x}", method))
+                    8 => {
+                        // DEFLATE
+                    }
```

Comment:
Seems unnecessary. In general I disagree with using line comments for things that aren't on their own line.